quickwit/rest-api-tests/scenarii/es_compatibility/0016-misc-query.yaml

```diff
@@ -87,7 +87,7 @@ json:
 expected:
   hits:
     total:
```

Comment:
Can we add an integration test where we see what happens if we target a field that is not a fast field?